packages/lib-classifier/src/components/Classifier/components/Layout/components/shared/StyledContainers.js

```diff
@@ -25,7 +11,21 @@ export const ViewerGrid = styled(Grid)`
 
 export const StyledSubjectContainer = styled(Box)`
   grid-area: subject;
-  position: sticky;
+  height: fit-content;
+
+  ${props => props.hasSurveyTask
```

Comment:
I'll add the `hasSurveyTask` property to the instances of `StyledSubjectContainer` and remove it from `ViewerGrid` in `lib-classifier_survey-layout-fixes`.